repo/repo.go

```diff
@@ -223,15 +224,16 @@ func (repo *Repo) GetRepo() map[string]*content.RepoNode {
 // reads the JSON history file from the Filesystem and copies it directly in to the supplied buffer
 // the result is wrapped as service response, e.g: {"reply": <contentData>}
 func (repo *Repo) WriteRepoBytes(w io.Writer) {
-
 	f, err := os.Open(repo.history.getCurrentFilename())
 	if err != nil {
 		Log.Error("failed to serve Repo JSON", zap.Error(err))
+		return
 	}
+	defer f.Close()
 
-	w.Write([]byte("{\"reply\":"))
-	_, err = io.Copy(w, f)
-	if err != nil {
+	w.Write([]byte(`{"reply":`))
+	// bufio.NewReaderSize speeds up reading in an AWS/GCP environment
+	if _, err := io.Copy(w, bufio.NewReaderSize(f, 2*4096)); err != nil {
```

Comment:
-> bufio.NewReaderSize speeds up reading in an AWS/GCP environment

Do you have benchmarks for this?

Reply:
I don;t have access to AWS/GCP but this is what the other bestbytes dev told me.
Also read here: https://www.datadoghq.com/blog/aws-ebs-provisioned-iops-getting-optimal-performance/